src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/RequestStatus.java

```diff
@@ -0,0 +1,9 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+public enum RequestStatus {
+    OPEN,
+    CHIPS_INCOMING_CONTACT_SENT,
```

Comment:
we don't know how many contacts will be sent to chips yet so just have OPEN, SUBMITTED, PROCESSED for the time being.